frontend/src/Metamaps/GlobalUI/Notifications.js

```diff
@@ -0,0 +1,54 @@
+import GlobalUI from './index'
+
+const Notifications = {
+  notifications: null,
+  unreadNotificationsCount: 0,
```

Comment:
it would be great if this was a function rather than a global variable like this. isn't that doable?

Reply:
You mean unreadNotificationsCount

Reply:
?

Reply:
Yeah

Reply:
@devvmh a function how? and where does the value get stored? 
locally, and then just return it within the function?

Reply:
```
unreadNotificationsCount: (notifications) => {
  return notifications.filter(n => !n.is_read).length
},
```

Reply:
Does that work? I just want to avoid storing global state

Reply:
no, that doesn't work because the notifications array can't be depended on for completeness. 
at any given time only a certain page of all the notifications may be stored locally. That's why we had the number occurring separately and coming down from the server

Reply:
Ok,  too bad.  Maybe I'll take a look sometime